test/utils.test.ts

```diff
@@ -744,6 +744,32 @@ describe('#organizeArguments()', () => {
 			utils(settings).organizeArguments();
 		}).toThrow();
 	});
+
+	test('Complains about unexpected pass-through arguments ', () => {
+		const settings = {
+			...settingsPizzaOrdering,
+			arguments: ['order', 'dine-in', '--', '--pass-through-flag', 'pass-through-option=true', 'pass-through-data'],
```

Comment:
This is a better representation of what an option is:

```suggestion
			arguments: ['order', 'dine-in', '--', '--pass-through-flag', 'pass-through-option=value', 'pass-through-data'],
```